packages/circuit-ui/components/Input/Input.tsx

```diff
@@ -355,8 +369,16 @@ export const Input = forwardRef(
         as={label ? 'label' : 'div'}
         css={labelStyles}
       >
-        {label && <LabelText hideLabel={hideLabel}>{label}</LabelText>}
-
+        {label && (
+          <LabelText hideLabel={hideLabel}>
+            {label}
+            {optionalLabel && !required ? (
+              <OptionalLabel>{` (${optionalLabel})`}</OptionalLabel>
+            ) : (
+              ''
+            )}
```

Comment:
Maybe we could make this `null` to avoid the empty line in all snapshots

Reply:
Somehow I had forgotten what to use when nothing should be rendered 🤦🏻‍♂️